src/igvfd/schemas/changelogs/measurement_set.md

```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`measurement_set.json`*
 
+### Schema version 13
+
+* Renamed `histone ChIP-seq`, `Parse Split-seq`, `Saturation genome editing`, `SHARE-Seq`, and `Yeast two-hybrid` to `Histone ChIP-seq`, `Parse SPLiT-seq`, `SGE`, `SHARE-seq` and `Y2H` respectively in preferred_assay_title.
```

Comment:
does it comply with the idea presented by Jennifer yesterday?

Reply:
we did not get to this case yesterday. Jennifer and I will talk more about this during wrangler's meeting tomorrow.